### PR TITLE
fix(cli): lhci open to process multiple urls

### PR DESCRIPTION
**Pull request by colourgarden**
### Description
- `lhci open` command currently silently fails if configuration `ci.collect.url` is an array of URLs (valid) rather than a string

### To reproduce
1. Set configuration:
  ```
  ci: {
    collect: {
      url: [
        'https://www.google.com/',
        'https://www.google.de/'
      ]
    }
  }
  ```
2. Run `lhci collect` to generate reports successfully
3. Run `lhci open`. No URL opened in browser (due to `continue`) but `Done!` message still printed to console

### CODE REVIEW

**Comment by colourgarden** [comment]
@googlebot I signed it!

File: packages/cli/src/open/open.js
Comment (patrickhulce) [inline packages/cli/src/open/open.js:42]: how about a target URLs set? I think this will fail when url is a string, not an array

`const targetUrls = new Set([].concat(options.url || []));`

```suggestion
    if (targetUrls.size && !targetUrls.has(lhr.finalUrl)) continue;
```

**Review by patrickhulce — CHANGES_REQUESTED** [review]
thanks very much @colourgarden!

this is mostly an accident for a different reason. the config isn't supposed to apply to the `open` command, but it seems reasonable to support anyhow.

File: packages/cli/src/open/open.js
Comment (colourgarden) [reply packages/cli/src/open/open.js:42]: @patrickhulce Thanks for the swift feedback! I did actually test my solution with both string and array and it does work with both. However, I'd defer to you as creator/maintainer on which approach is best. I've quickly tested your solution locally and it also works as expected 👍 

I assume `targetUrls` is best set on line 40 after the `representativeLhrs` check?

Comment (patrickhulce) [reply packages/cli/src/open/open.js:42]: > it does work with both

Only when the `finalUrl` of all the reports matches `options.url` ;) `TypeError: options.url.some is not a function` if you have an LHR with a non-matching URL that should be skipped and `options.url` is a string

> I assume targetUrls is best set on line 40 after the representativeLhrs check?

Yep SGTM, thanks! 👍

**Review by patrickhulce — APPROVED** [review]
thanks @colourgarden!

File: packages/cli/src/open/open.js
Comment (patrickhulce) [inline packages/cli/src/open/open.js:41]: aw shoot, sorry @colourgarden but typescript doesn't like my too clever suggestion 😅 
```suggestion
  const targetUrls = typeof options.url === 'string' ? 
    [options.url] : 
    (options.url || []);
```

File: packages/cli/src/open/open.js
Comment (patrickhulce) [inline packages/cli/src/open/open.js:44]:
```suggestion
    if (targetUrls.length && !targetUrls.includes(lhr.finalUrl)) continue;
```

**Review by patrickhulce — COMMENTED** [review]
sorry for steering you in the wrong direction!

**Review by patrickhulce — APPROVED** [review]

**Comment by patrickhulce** [comment]
@googlebot I consent.

**Pull request merged**
